.gitignore

```diff
@@ -23,3 +23,9 @@ coverage
 *.njsproj
 *.sln
 *.sw?
+
+# Synpress directories and files
+.env
+tests/e2e/screenshots
+tests/e2e/videos
```

Comment:
why not all screenshots and videos? what if someone moves those directories, they'll have to know to update this.

Reply:
@turadg Updated

Reply:
That's still not all screenshots and videos. I don't understand the expectation that some screenshots or videos would ever be committed to this repo but given that I've asked three times not to have `tests` in gitignore, I assume you have a strong opinion about retaining it.